Kentico.Kontent.Management/Models/Shared/AssetWithRenditionsReference.cs

```diff
@@ -5,29 +5,54 @@
 
 namespace Kentico.Kontent.Management.Models.Shared;
 
+/// <summary>
+/// Represents identifier of asset with renditions.
+/// </summary>
 public sealed class AssetWithRenditionsReference
 {
     private readonly IList<Reference> _renditions;
 
     private readonly Reference _assetReference;
 
+    /// <summary>
+    /// Gets the id of the asset identifier.
+    /// </summary>
     [JsonProperty("id", DefaultValueHandling = DefaultValueHandling.Ignore)]
     public Guid? Id => _assetReference.Id;
 
+    /// <summary>
+    /// Gets the codename of the asset identifier.
+    /// </summary>
     [JsonProperty("codename", DefaultValueHandling = DefaultValueHandling.Ignore)]
     public string Codename => _assetReference.Codename;
 
+    /// <summary>
+    /// Gets the external id of the asset identifier.
+    /// </summary>
     [JsonProperty("external_id", DefaultValueHandling = DefaultValueHandling.Ignore)]
     public string ExternalId => _assetReference.ExternalId;
 
+    /// <summary>
+    /// Gets identifiers of linked renditions.
+    /// </summary>
     [JsonProperty("renditions", DefaultValueHandling = DefaultValueHandling.Ignore)]
     public IEnumerable<Reference> Renditions => _renditions;
 
+    /// <summary>
+    /// Creates instance from asset reference and a single rendition reference.
+    /// </summary>
+    /// <param name="assetReference">Asset reference.</param>
+    /// <param name="renditionReference">Rendition reference.</param>
     public AssetWithRenditionsReference(Reference assetReference, Reference renditionReference)
         : this(assetReference, new[] { renditionReference })
     {
     }
 
+    /// <summary>
+    /// Creates instance from asset reference and a possibly multiple rendition reference.
```

Comment:
"...and multiple rendition references" (your formulation does not make sense)

Reply:
The property is nullable. Therefore "possibly" :D

Reply:
...and multiple rendition references (if present)"? :D